src/reconcile/rec_write.c

```diff
@@ -1987,10 +1987,9 @@ __rec_split(WT_SESSION_IMPL *session, WT_RECONCILE *r, size_t next_len)
 		 * have one.  If we don't have room for another split chunk,
 		 * add whatever space remains in this page.
 		 */
-		if (inuse + r->split_size <= r->page_size)
-			r->space_avail =
-			    r->split_size - WT_PAGE_HEADER_BYTE_SIZE(btree);
-		else {
+		r->space_avail =
+		    r->split_size - WT_PAGE_HEADER_BYTE_SIZE(btree);
+		if (inuse + r->space_avail > r->page_size) {
```

Comment:
Right... the correct test for "will a split-chunk fit" is:
`if (inuse + (split_size - header)) <= page_size)`

OK, I believe that.